.github/workflows/docker.yml

```diff
@@ -5,43 +5,37 @@ name: Docker
 #
 # Summary
 #
-# job docker-branch-os-matrix:
-# * creates tags latest-alpine, latest-debian and latest-ubuntu for main branch
-# * creates tags stable-alpine, stable-debian and stable-ubuntu for releasebranch_8_2
-# * creates tags <branch_name>-alpine, <branch_name>-debian and <branch_name>-ubuntu for all triggered branches
-#
-# job docker-main-latest:
-# * creates tag latest for main branch
-#
-# job docker-release-os-matrix:
+# job docker-os-matrix:
 # * creates tags <version>-alpine, <version>-debian and <version>-ubuntu for each release
+# * creates tags <branch_name>-alpine, <branch_name>-debian and <branch_name>-ubuntu for all triggered branches
+# * creates tags current-alpine, current-debian and current-ubuntu for releasebranch_8_3
+# * creates tag latest for last stable release with ubuntu os
 
 on:
   push:
     branches:
       - main
       - releasebranch_*
       - '!releasebranch_7_*'
-    tags: ['*.*.*']
+    # tags: ['*.*.*']
```

Comment:
I'm not sure about removing tags. The tags are created later on after a commit was made and pushed, so when the branch is updated with a commit, there is no tag. The tag is added only later. Will this be triggered? It is not clear to me if your tests included tags or not.

Reply:
My intention was to simplify things and only have docker builds as described above. I also tested that case.
In these changes a tag itself is not triggered, so e.g. `8.3.0RC1-ubuntu` would not be build and tagged anymore, only the published releases like `8.3.0`. This also prevents the build from running twice on a release. Before for each published release the docker images were build two times - once for the tag and once for the release itself. This might not happen now with the simplification into a single workflow, but I have tested only the case without the tags.
So do you think also pre-releases are important to be build and tagged on dockerhub? Then I can readd this line.

Reply:
I missed that release part. Thanks for the explanation.

Reply:
And no, I don't think not having RCs is a problem. However, I think these are still published releases, so they fulfill the condition or are they excluded because they are pre-releases? It seems that pre-releases are to excluded from released but not from published ([doc](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release)):

> Note: The prereleased type will not trigger for pre-releases published from draft releases, but the published type will trigger. If you want a workflow to run when stable and pre-releases publish, subscribe to published instead of released and prereleased.

Reply:
Ah ok then, so `published` simply sorts out the `draft` state of the releases. Thanks for diving into the docs :)
I don't have a strong opinion on having the RCs be build and pushed or not. As that doesn't happen too often, I am also fine with keeping them. Would you prefer only type `released` or we leave it as is?

Reply:
Leave it as is. Having the RCs makes sense, you can easily test the release candidate and testing is certainly what RCs are meant for.